pkg/api/types.go

```diff
@@ -505,6 +525,11 @@ type ImageSourcePath struct {
 type RPMImageInjectionStepConfiguration struct {
 	From PipelineImageStreamTagReference `json:"from"`
 	To   PipelineImageStreamTagReference `json:"to,omitempty"`
+
+	// AlwaysDumpBuildLog let's you choose to dump the OpenShift Docker
+	// build log regardless of success or failure.  The default is to
+	// only dump the build log on a failure.
+	AlwaysDumpBuildLog bool `json:"always_dump_build_log"`
```

Comment:
@stevekuznetsov, should all these be `omitempty` so that we do not need to write all those falses next time we need to serialize ciop configs?

Reply:
Good clal